### ISSUE TITLE
Texture errors and weapon switching brakes when loading from save with enhanced textures, models, fx

### ISSUE THREAD

**Issue by Leumeister** [shinyquagsire23/OpenJKDF2#218]
I recently installed the Jedi Knight Enhanced mods into OpenJKDF2 manually, and the game seems to work fine when loading a new game, but loading from a save, whether a quicksave or a menu save causes the wrong textures to be loaded and applied to the level geometry, and in some cases, no weapon to load.

At one point, I loaded into a save in the refuelling station, and not only were the textures wrong, but in place of Kyle's Bryar pistol was an invisible item spawned pipes at wherever the camera was aiming, with full collision.

This is in the hallway near the level exit in Level 12 - Escape with the Map
![20230202162638_1](https://user-images.githubusercontent.com/41119963/216277125-c59d0ff4-cd04-44ae-92ae-048b61ff3341.jpg)

**Comment by shinyquagsire23** [comment]
Yeah this is a known quirk with JKDF2 saves unfortunately, if a mod edits static.jkl or otherwise shifts the texture indices, it causes old saves to load incorrectly. At some point I'd like to find a good way to fix this, or at least provide some enhanced savegame information to mitigate bugs like this with mods. Installing COG mods and then loading saves can also cause some pretty weird stuff.

**Comment by Leumeister** [comment]
Ahhh, yeah, I noticed there was a texture glitch in Vanilla JKDF2 at one point as well, but I haven't been able to reproduce it. It didn't break anything else, though; it was more like there was a palette issue or something, like the level directly after the first fight with Yun got a blue palette from the level before it.
